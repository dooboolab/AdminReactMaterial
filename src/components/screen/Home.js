import { inject, observer } from 'mobx-react';
import React, { Component } from 'react';

export class Home extends Component<any, any> {
  constructor(props) {
    super(props);
    this.state = {
    };
  }

  render() {
    return (
      <div style={{ flex: 1 }}>
        Home
      </div>
    );
  }
}

const defaultHome = inject('store')(observer(Home));
export default defaultHome;
