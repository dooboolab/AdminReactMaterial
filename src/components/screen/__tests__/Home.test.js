import React from 'react';
import { shallow, render } from 'enzyme';

import appStore from '../../../stores/appStore';
import Home from '../Home';

// test for the container page in dom
describe('Home page DOM rendering test', () => {
  const page = shallow(
    <Home store={appStore} />,
  );

  it('Tab1 page has to be matched to snapshot', () => {
    expect(page).toMatchSnapshot();
  });
});
