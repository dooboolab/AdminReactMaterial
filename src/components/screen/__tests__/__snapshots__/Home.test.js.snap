// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page DOM rendering test Tab1 page has to be matched to snapshot 1`] = `
<Home
  store={[Function]}
/>
`;
